Formula/fuseki.rb

```diff
@@ -1,11 +1,12 @@
 class Fuseki < Formula
   desc "SPARQL server"
   homepage "https://jena.apache.org/documentation/serving_data/"
-  url "https://archive.apache.org/dist/jena/binaries/jena-fuseki1-1.1.2-distribution.tar.gz"
-  version "1.1.2"
-  sha256 "78bd92b4e32f9e918d89946d11aed9789416f4058b127af60b251b4a8636b5f0"
+  url "https://archive.apache.org/dist/jena/binaries/jena-fuseki1-1.4.0-distribution.tar.gz"
+  version "1.4.0"
+  sha256 "8f0cfa13d9a94df9dfb5a424177a0d3d9873b605e0ae610ba6e5f3d30f06f9bf"
 
   bottle :unneeded
+  plist_options :startup => true
```

Comment:
Yeah, you're right. I think I can just remove `plist_options :startup => true` as long as user will can this message below:
```
...
To have launchd start fuseki now and restart at login:
  brew services start fuseki
```
Am I right?